superdesk/publish/publish_content.py

```diff
@@ -78,9 +79,18 @@ def transmit_items(queue_items):
             logger.exception(ex)
             failed_items[str(queue_item.get('_id'))] = queue_item
 
+    # mark failed items as pending so that Celery tasks will try again
     if len(failed_items) > 0:
         for item_id in failed_items.keys():
-            get_resource_service(PUBLISH_QUEUE).system_update(item_id, {'state': STATE_PENDING}, failed_items[item_id])
+            try:
+                orig_item = publish_queue_service.find_one(item_id)
+                publish_queue_service.system_update(
+                    item_id, {'state': STATE_PENDING}, orig_item)
+            except:
+                logger.error(
```

Comment:
Does the traceback matter here?
I can send another PR to address this tomorrow.